doc/users/next_whats_new/2017-12-05_cividis_colormap.rst

```diff
@@ -0,0 +1,13 @@
+Cividis colormap
+----------------------------
+
+A new colormap, named cividis, has been optimized so that it looks similar to people with a red-green color vision deficiency. For more information, please see our paper (to be published in 2018).
```

Comment:
This now renders fine.  https://4348-1385122-gh.circle-artifacts.com/0/home/circleci/project/doc/build/html/users/next_whats_new/2017-12-05_cividis_colormap.html

... but, "please see our paper" is too mysterious!  "Our paper" will typically mean Matplotlib's paper, and this has no way of knowing what paper you mean.